Software/Development/SemesterProject/App/Include/dutyCycle.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * dutyCycle.h
+ *
+ *  Created on: 12 mar. 2019
+ *      Author: Aitor Teran
+ */
+
+#ifndef APP_DUTYCYCLE_H_
+#define APP_DUTYCYCLE_H_
+
+
+//
+//PIN definition. First item is the pwm to be outputed and
+//the second item is the EPWM pin where the connector is placed in the PCB.
+//It is in hexadecimal to allow correct understanding of each pin.
+//CONSIDER THE LOCATION OF THESE DEFINITIONS AS WELL AS THE HEXADECIMAL FOR REVIEW.
+//
+#define PWM_A   0xA1
+#define PWM_B   0xA2
+#define PWM_C   0xA3
+
+
+//
+// Object declaration
+//
+typedef struct DutyCycleTag
+{
+    int dutyValue;
+    int pin
+} DutyCycle;
+
+
+//
+// Function prototyping
+//
+void setDutyCycleA(float duty);
+void setDutyCycleB(float duty);
+void setDutyCycleC(float duty);
```

Comment:
Have you considered just creating one method for setting the duties (instead of A, B and C) and then just pass both the duty and the specific leg?

Reply:
I did consider that but i thought that it could be easier to understand this way. I thought it would also lead to easier implementation of future code.